content/config/filter.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Filter command outputs
```

Comment:
```suggestion
title: Filter commands
```

This isn't only about output. This page focuses on output/printing throughout. Need to generalize - it's also about actions, like prune.